Retail and SLE15 Images update (#303)
uyuni-project/uyuni-docs: modules/salt/pages/formula-saltboot.adoc

Leave the other fields empty.

[[saltboot-formula-disk-selection]]

> == Disk selection in saltboot formula

Reviewer: ```suggestion
== Disk Selection in Saltboot Formula
```

> When there is only one disk present on target hardware (including USB drives), an asterisk [systemitem]``*`` can be used to automatically select the disk device.

Reviewer: ```suggestion
When there is only one disk present on target hardware (including USB drives), use an asterisk [systemitem]``*`` to automatically select the disk device.
```

> When there are multiple disks, asterisk [systemitem]``*`` can be used in device path to differentiate i.e. SATA disks from USB disks:

Reviewer: ```suggestion
When there are multiple disks, use an asterisk [systemitem]``*`` in the device path.
In this example, SATA disks are differentiated from USB disks:
```

----
/dev/disk/by-path/*-ata-1
/dev/disk/by-path/*usb*
----

> If the entered value does not contain "[systemitem]``/``", the entered value is automatically prepended by "[path]``/dev/disk/by-path/``". That means that i.e. [systemitem]``\*usb*`` is the same as [path]``/dev/disk/by-path/\*usb*``.

Reviewer: ```suggestion
If the entered value does not contain "[systemitem]``/``", the entered value is automatically prepended by "[path]``/dev/disk/by-path/``". 
For example, [systemitem]``\*usb*`` is the same as [path]``/dev/disk/by-path/\*usb*``.
```

Reviewer: As an aside: "ie" means "that is", and "eg" means "for example". Moot point, though, since we don't use Latin abbreviations in our docs as a rule ;)

Author: :facepalm: I always thought they are 'example given' and 'in example'. Now that is embarrassing. :smile:

> If selecting exact devices is preferred, disk device field can be in format of:

Reviewer: ```suggestion
If you prefer to select specific devices, you can this format in the disk device field:
```

* symbolic names (e.g. [path]``/dev/sda``)
* by-path (e.g. [path]``/dev/disk/by-path/..``)
* by-id (e.g. [path]``/dev/disk/by-id/...``)

> List of available devices is visible on the terminal if you press [systemitem]``ESC`` during waiting for key approval.

Reviewer: ```suggestion
To see a list of available devices from the command prompt, press kbd:[Esc] while waiting for key approval.
```

Leave the other fields empty.

[[saltboot-formula-disk-selection]]
== Disk selection in saltboot formula

When there is only one disk present on target hardware (including USB drives), an asterisk [systemitem]``*`` can be used to automatically select the disk device.

If selecting exact devices is preferred, disk device field can be in format of symbolic names (e.g. [path]``/dev/sda``), by-path (e.g. [path]``/dev/disk/by-path/..``) or by-id (e.g. [path]``/dev/disk/by-id/...``)

> 

Reviewer 2: /dev/disk/by-path/ prefix is used by default, if the name does not contain any /
so /dev/disk/by-path/*usb* is the same as *usb*

List of available devices is visible on the terminal if you press ESC during waiting for key approval.

Author: Updated in latest commit.
